wolfcrypt/src/asn.c

```diff
@@ -2316,8 +2324,11 @@ int ToTraditionalInline_ex(const byte* input, word32* inOutIdx, word32 sz,
     }
 
     ret = GetOctetString(input, &idx, &length, sz);
-    if (ret < 0)
-        return ret;
+    if (ret < 0) {
+        /* Don't error out if return is error - some private keys do not expect
```

Comment:
Maybe so, but BUFFER_E is a real error, and idx and length may not be valid values but they're used as if they are now.